src/main/java/Duke.java

```diff
@@ -1,10 +1,50 @@
+import java.util.*;
```

Comment:
I think the import statement can be more specific?
[https://se-education.org/guides/conventions/java/basic.html#statements](url)